### ISSUE TITLE
It is still slow in dppg.py, however, it's quite normal in pre_train.py,why?

### ISSUE THREAD

**Issue by lyl2854** [QiyuZ/auto_car_experience_training#2]
May I use the trained data to run the ddpg.py?

**Comment by QiyuZ** [comment]
Yes, pre_train is to train the driving example (an existing driving strategy offered by original author), and the model is based on that. So think about that, the ddpg is train the car total from scratch but this is based on a example strategy and then to train it.

**Comment by BCWang93** [comment]
> Yes, pre_train is to train the driving example (an existing driving strategy offered by original author), and the model is based on that. So think about that, the ddpg is train the car total from scratch but this is based on a example strategy and then to train it.

@QiyuZ  Hello,I am very insterested in your project.I want to know which code is the 'offline learning code',and which is 'inline learning code'.what diffierence between 'ddpg.py' and 'ddpg_tune.py'?.Thank you very much!Thanks!
